src/Illuminate/Routing/Router.php

```diff
@@ -296,6 +296,21 @@ public function group(array $attributes, $routes)
     }
 
     /**
+     * Create a redirect from one route to another.
+     *
+     * @param string $url
+     * @param string $destination
+     * @param int $status
+     * @return Route
+     */
+    public function redirect($url, $destination, $status = 301)
+    {
+        return $this->any($url, '\Illuminate\Routing\RedirectController')
```

Comment:
I checked a couple other cases and the core and strings seemed to be the convention. I'm happy to use class notation, Taylor can always change it if he's not happy with it.